tests/command_line/test_version.py

```diff
@@ -652,6 +676,107 @@ def test_version_runs_build_command(
+        "TMP": "C:\\Users\\Username\\AppData\\Local\\Temp",
+        "USERPROFILE": "C:\\Users\\Username",
+        "USERSID": "S-1-5-21-1234567890-123456789-123456789-1234",
+        "WINDIR": "C:\\Windows"
+    }
+
+    # Mock out subprocess.run
+    with mock.patch(
+        "subprocess.run", return_value=CompletedProcess(args=(), returncode=0)
+    ) as patched_subprocess_run, mock.patch(
+        "shellingham.detect_shell", return_value=(exe, shell)
+    ), mock.patch(
+        "sys.platform", "win32"
+    ), mock.patch.dict("os.environ", patched_os_environment, clear=True):
+        # ACT: run & force a new version that will trigger the build command
+        result = cli_runner.invoke(
+            main, [version_subcmd or "version", "--patch", "--no-push"]
+        )
+        assert result.exit_code == 0
+
+        patched_subprocess_run.assert_called_once_with(
+            [exe, "-c" if shell != "cmd" else "/c", build_command],
+            check=True,
+            env={
+                "NEW_VERSION": "1.2.1",  # injected into environment
+                "CI": patched_os_environment["CI"],
+                "BITBUCKET_CI": "true",  # Converted
+                "GITHUB_ACTIONS": patched_os_environment["GITHUB_ACTIONS"],
+                "GITEA_ACTIONS": patched_os_environment["GITEA_ACTIONS"],
+                "GITLAB_CI": patched_os_environment["GITLAB_CI"],
+                "HOME": patched_os_environment["HOME"],
+                "PATH": patched_os_environment["PATH"],
+                "VIRTUAL_ENV": patched_os_environment["VIRTUAL_ENV"],
+                "PSR_DOCKER_GITHUB_ACTION": patched_os_environment[
+                    "PSR_DOCKER_GITHUB_ACTION"
+                ],
+                # Windows
+                "ALLUSERSAPPDATA": "C:\\ProgramData",
+                "ALLUSERSPROFILE": "C:\\ProgramData",
+                "APPDATA": "C:\\Users\\Username\\AppData\\Roaming",
+                "COMMONPROGRAMFILES": "C:\\Program Files\\Common Files",
+                "COMMONPROGRAMFILES(x86)": "C:\\Program Files (x86)\\Common Files",
+                "DEFAULTUSERPROFILE": "C:\\Users\\Default",
+                "HOMEPATH": "\\Users\\Username",
+                "PATHEXT": ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC",
+                "PROFILESFOLDER": "C:\\Users",
+                "PROGRAMFILES": "C:\\Program Files",
+                "PROGRAMFILES(x86)": "C:\\Program Files (x86)",
+                "SYSTEM": "C:\\Windows\\System32",
+                "SYSTEM16": "C:\\Windows\\System16",
+                "SYSTEM32": "C:\\Windows\\System32",
+                "SYSTEMDRIVE": "C:",
+                "SYSTEMROOT": "C:\\Windows",
+                "TEMP": "C:\\Users\\Username\\AppData\\Local\\Temp",
+                "TMP": "C:\\Users\\Username\\AppData\\Local\\Temp",
+                "USERPROFILE": "C:\\Users\\Username",
+                "USERSID": "S-1-5-21-1234567890-123456789-123456789-1234",
+                "WINDIR": "C:\\Windows"
```

Comment:
If you could have all these reference the `patched_os_environment["VAR_NAME"]` so that it is easier to maintain and it is congruent with the rest of it, that would be great!